torchtune/models/clip/_transforms.py

```diff
@@ -130,11 +221,23 @@ def __init__(
 
         # resize_with_pad
         self.max_upscaling_size = None if resize_to_max_canvas else tile_size
-        self.resample = torchvision.transforms.InterpolationMode[resample.upper()]
+        self.resample = resample
 
         # tile_crop
         self.tile_size = tile_size
 
+        # image size upper bound for export
```

Comment:
same comment here about export. Maybe just doing torch.export makes it a bit more intuitive for the user, who has no clue that exporting is a thing